### DISCUSSION TITLE
This driver does not support creating temporary URLs.

### DISCUSSION THREAD

**Discussion by ShekhSaifuddin007** [livewire/livewire#2067] [1. Help]
How can I solve this issue..? when I upload a file like `.pdf` or something which not image it shows me this error

`ErrorException
This driver does not support creating temporary URLs. (View: G:\xampp\htdocs\pos-client\resources\views\livewire\products\create.blade.php)`

### my markup

```blade
@if ($image)
    <img src="{{ $image->temporaryUrl() }}" alt="Product Image" class="w-100 rounded" style="height: 140px">
@else
   <div class="image-preview">
      <span>Preview Image</span>
  </div>
@endif
```

**Comment by joshhanley** [comment]
You can't use temporary urls for files, see here in the docs https://laravel-livewire.com/docs/2.x/file-uploads#preview-urls
> Note: for security reasons, temporary urls are only supported for image uploads.

Hope this helps!

**Comment by mblarsen** [comment]
Related #1106
